src/a2ln/a2ln.py

```diff
@@ -235,7 +240,7 @@ def replace(text: str) -> str:
                                      daemon=True).start()
 
                     if self.command is not None:
-                        subprocess.Popen(replace(self.command), shell=True)
+                        subprocess.Popen(replace(self.command), shell=False)
```

Comment:
@yozachar Wouldn't  `shell=False` break some things?  https://docs.python.org/3/library/subprocess.html#frequently-used-arguments shows that `shell=True` allows you to use shell features such as pipes. Isn't this very useful?

Reply:
https://github.com/patri9ck/a2ln-server/blob/bfb2889257e510a7b38bad984e51e25d33707ba8/src/a2ln/a2ln.py#L119-L120

@patri9ck, command passed as values to `--command` gets executed as a sub-process, right?

When you set `shell=True` (and pass in, the user provided command, to shell), you're opening up your code to some potential security holes. Ref: https://docs.python.org/3/library/subprocess.html#security-considerations

A majority of  simple commands can be executed with `shell=False`, I recommend that you keep it so and when a specific and strong use-cases comes up, you can conditionally set `shell=True`.

Also if you do need to use shell pipes, you can do so:

```py
import subprocess
p1 = subprocess.Popen(["cat", "file.log"], stdout=subprocess.PIPE)
p2 = subprocess.Popen(["tail", "-1"], stdin=p1.stdout, stdout=subprocess.PIPE)
p1.stdout.close()  # Allow p1 to receive a SIGPIPE if p2 exits.
output,err = p2.communicate()
```

Ref: https://stackoverflow.com/a/7353141